README.md

```diff
@@ -2,4 +2,43 @@
 
 3D Slicer Extension for Openwater's OpenLIFU project
 
-Build this extension by following [the usual procedure for Slicer extensions](https://slicer.readthedocs.io/en/latest/developer_guide/extensions.html#build-an-extension).
\ No newline at end of file
+Build this extension by following [the usual procedure for Slicer extensions](https://slicer.readthedocs.io/en/latest/developer_guide/extensions.html#build-an-extension).
+
+## Pairing with 3D Open Water App
+
+### Install Android Platform Tools
+
+**macOS:**  
+
+```bash
+brew install android-platform-tools
+```
+
+**Linux:**  
+
+```bash
+sudo apt update
+sudo apt install android-tools-adb
+```
+
+**Windows (PowerShell):**  
+
+```powershell
+Set-ExecutionPolicy RemoteSigned -Scope CurrentUser
+irm get.scoop.sh | iex
+scoop install android-platform-tools
+```
+
+Or download from [Google's
+platform-tools](https://developer.android.com/tools/releases/platform-tools) and
+add it to your `PATH`.
+
+### Enable USB Debugging on Android
+
+1. On your Android device, go to **Settings → About phone**.
```

Comment:
I had to also go to "Software information" after "About phone" to find "build number"